cmd/spire-server/cli/run/run.go

```diff
@@ -35,16 +35,19 @@ type runConfig struct {
 }
 
 type serverConfig struct {
-	BindAddress   string `hcl:"bind_address"`
-	BindPort      int    `hcl:"bind_port"`
-	BindHTTPPort  int    `hcl:"bind_http_port"`
-	TrustDomain   string `hcl:"trust_domain"`
-	LogFile       string `hcl:"log_file"`
-	LogLevel      string `hcl:"log_level"`
-	BaseSVIDTtl   int    `hcl:"base_svid_ttl"`
-	ServerSVIDTtl int    `hcl:"server_svid_ttl"`
-	ConfigPath    string
-	Umask         string `hcl:"umask"`
+	BindAddress      string `hcl:"bind_address"`
+	BindPort         int    `hcl:"bind_port"`
+	BindHTTPPort     int    `hcl:"bind_http_port"`
+	TrustDomain      string `hcl:"trust_domain"`
+	LogFile          string `hcl:"log_file"`
+	LogLevel         string `hcl:"log_level"`
+	BaseSVIDTtl      int    `hcl:"base_svid_ttl"`
+	ServerSVIDTtl    int    `hcl:"server_svid_ttl"`
+	ConfigPath       string
+	Umask            string `hcl:"umask"`
+	ProfilingEnabled string `hcl:"profiling_enabled"`
```

Comment:
Same comment here about bool and int types

Reply:
Same problem here...